src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
```

Comment:
```suggestion
* {
  box-sizing: border-box;
}
```

Reply:
Agreed with Viktor

Reply:
Thanks for the suggestions. 
I've added box-sizing: border-box to the body in the commits 497a8361cee2ecad9025122a261d5a3ac16db94d and 10382f7918d9effcd44b7a5de593421554c60af2 (in that case all tests are passed). 
When I use * {
  box-sizing: border-box;
}
some tests are failed